mm2src/mm2_test_helpers/dummy_files/iris_nimda_history.json

```diff
@@ -0,0 +1,391 @@
+        ],
+        "to": [],
+        "total_amount": "0.028433",
+        "spent_by_me": "0.028433",
+        "received_by_me": "0",
+        "my_balance_change": "-0.028433",
+        "block_height": 6148309,
+        "timestamp": 1669798875,
+        "fee_details": null,
+        "coin": "IRIS-NIMDA",
+        "internal_id": "0000000000000000323544394131463143393338334143374435364533453139",
+        "transaction_type": {
+            "CustomTendermintMsg": {
+                "msg_type": "FeeForTokenTx",
+                "token_id": "e13c266c61de5e79cf37a99b94d4d4462d6857fe8a5dbdde9e041ae01b14a7db"
+            }
+        }
+    },
+    {
+        "tx_hex": "0a2a696161317a36303967397a386566346a6c6b63666767333663326c6a67636d6e6135306c32796a726130122a696161316530727838376d646a37397a656a65777563346a6737716c39756432323836673275733866322a0c0a036e696d1205313030303032403738646661636564613462346362373336366362626638643436336237393061666566363566656538613764356464356264616633613365303766343330623640980c",
+        "tx_hash": "25D9A1F1C9383AC7D56E3E19F3D6B9ED3F52F7D0E9225ABBEA0258FF75E28FC2",
+        "from": [
+            "iaa1z609g9z8ef4jlkcfgg36c2ljgcmna50l2yjra0"
+        ],
+        "to": [],
+        "total_amount": "0.01",
+        "spent_by_me": "0.01",
+        "received_by_me": "0",
+        "my_balance_change": "-0.01",
+        "block_height": 6148309,
+        "timestamp": 1669798875,
+        "fee_details": {
+            "type": "Tendermint",
+            "coin": "IRIS-NIMDA",
+            "amount": "0.028433",
+            "gas_limit": 100000
+        },
+        "coin": "IRIS-NIMDA",
+        "internal_id": "3931453345363544374341333833394331463141394435320000000000000000",
+        "transaction_type": {
+            "CustomTendermintMsg": {
+                "msg_type": "SendHtlcAmount",
+                "token_id": "e13c266c61de5e79cf37a99b94d4d4462d6857fe8a5dbdde9e041ae01b14a7db"
+            }
+        }
+    },
+    {
+        "tx_hex": "0a2a696161317a36303967397a386566346a6c6b63666767333663326c6a67636d6e6135306c32796a726130122a696161316567307167617a37336a737676727676747a713478383233686d7a387161706c64643078347a1a0a0a036e696d1203313030",
+        "tx_hash": "D83D8143CAD8C225F3BC24C6DCA324DB2F63C97A21AE2C030328266E9415E50A",
+        "from": [
+            "iaa1z609g9z8ef4jlkcfgg36c2ljgcmna50l2yjra0"
+        ],
+        "to": [],
+        "total_amount": "0.025763",
+        "spent_by_me": "0.025763",
+        "received_by_me": "0",
+        "my_balance_change": "-0.025763",
+        "block_height": 6148307,
+        "timestamp": 1669798865,
+        "fee_details": null,
+        "coin": "IRIS-NIMDA",
+        "internal_id": "0000000000000000443833443831343343414438433232354633424332344336",
+        "transaction_type": {
+            "CustomTendermintMsg": {
+                "msg_type": "FeeForTokenTx",
+                "token_id": "e13c266c61de5e79cf37a99b94d4d4462d6857fe8a5dbdde9e041ae01b14a7db"
+            }
```

Comment:
Could you please clarify what's the difference between `FeeForTokenTx` and `TokenTransfer`?

Reply:
Also I'm afraid that such enumeration can be a bottleneck for the GUI to parse.
What do you think about
```rust
"transaction_type": {
  "type": "CustomTendermintMsg",
  "details": {
    "msg_type": "FeeForTokenTx",
    "token_id": "e13c266c61de5e79cf37a99b94d4d4462d6857fe8a5dbdde9e041ae01b14a7db",
  }
}
```
and
```rust
"transaction_type": {
  "type": "TokenTransfer",
  "details": {
    "token_id": "e13c266c61de5e79cf37a99b94d4d4462d6857fe8a5dbdde9e041ae01b14a7db"
  }
}
```

Reply:
> Could you please clarify what's the difference between FeeForTokenTx and TokenTransfer?

`TokenTransfer` is for where you transfer tokens.

`FeeForTokenTx` is for where you pay for fees of token transactions. Fees are paid in platform coin in cosmos. We wanted show tokens fees as as seperated transaction in tx history.


> What do you think about

Yeah looks good. But this will brake BC. Should I do it?

Reply:
Missed this... Let's leave it as it is then
> Yeah looks good. But this will brake BC. Should I do it?